dotcom-rendering/src/web/components/Callout/Form.stories.tsx

```diff
@@ -1,60 +1,50 @@
-import { css } from '@emotion/react';
-import {
-	calloutCampaign,
-	calloutCampaignOnlyTwoRadio,
-} from '../../../../fixtures/manual/calloutCampaign';
```

Comment:
Previously we had two different mock campaigns in the `fixtures` folder - I merged them so that one story could capture all the various form elements and to reduce code/story bloat.